volto/custom-widget.rst

```diff
@@ -0,0 +1,7 @@
+.. _custom_widget-label:
+
+=============
+Custom Widget
+=============
+
+time widget?
```

Comment:
This makes no sense to me. Is this just a WIP placeholder?

Reply:
yes